src/app/components/employee-profile/employee-profile.component.ts

```diff
@@ -1,15 +1,44 @@
+import { HttpClient } from '@angular/common/http';
 import { Component, OnInit } from '@angular/core';
+import { Router, ActivatedRoute, ParamMap } from '@angular/router'
 
 @Component({
-  selector: 'app-perfil-empleado',
+  selector: 'app-employee-profile',
   templateUrl: './employee-profile.component.html',
   styleUrls: ['./employee-profile.component.scss']
 })
 export class EmployeeProfileComponent implements OnInit {
 
-  constructor() { }
+  id: any;
+  users = []
+
+  constructor(private http: HttpClient){}
+
+  loadUser(){
+    this.http
+    .get('mongodb://mongo:27017/yuva')
```

Comment:
Aca no deben de haber conexiones a la base de datos desde el front, es una vulnerabilidad muy peligrosa conectarse desde aquí, siempre hay que manejar todo la información desde el Back

 https://stackoverflow.com/questions/29699753/why-is-it-a-bad-practice-to-connect-a-sql-database-from-javascript